### ISSUE TITLE
Bootstrap 3 and 4 support

### ISSUE THREAD

**Issue by kingcody** [third774/ng-bootstrap-form-validation#28]
Looking over the final style classes for bs4 I believe it would be possible to support both version 3 and 4 simultaneously out-of-the-box. I'd like to discuss if this is something that should be done. While it might not be the cleanest and most general approach, this is a bootstrap themed validation library so I don't think it's completely out of scope.

@third774?

**Comment by third774** [comment]
My preference would be to not be adding multiple classes from multiple versions, but have this be a configuration option that is provided at the top level. We could either accept a string and just use that, or perhaps an enum?

Beyond that - what should the default be? I think we should default to v4.

Thoughts?

**Comment by AmirGilboa** [comment]
I agree. the default should be the latest version. support for previous version(s) is a nice feature, but less important (imho) than keeping up with the latest. Those who are working with bs3 can simply use previous versions of this component.
